paddle/framework/tensor.h

```diff
@@ -0,0 +1,76 @@
+/*
+  Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
The copyright message has a different style with [formal](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/gserver/layers/AddtoLayer.h#L1)，what format should we use in the newly added code?